support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -258,12 +260,32 @@ export function ThreeTierLanding(): JSX.Element {
 				cssOverrides={recurringContainer}
 			>
 				<div css={innerContentContainer}>
-					<h1 css={heading}>
-						Support fearless, <br css={tabletLineBreak} />
-						independent journalism
+					<h1 css={headlineCss}>
+						<picture>
+							<source
+								media="(max-width: 739px)"
+								srcSet="https://uploads.guim.co.uk/2024/01/17/headlineMobile.svg"
+								type="image/svg+xml"
+							/>
+							<source
+								media="(max-width: 979px)"
+								srcSet="https://uploads.guim.co.uk/2024/01/17/headlineTablet.svg"
+								type="image/svg+xml"
+							/>
+							<source
+								srcSet="https://uploads.guim.co.uk/2024/01/17/headlineDesktop.svg"
+								type="image/svg+xml"
+							/>
+
+							<img
+								src="https://uploads.guim.co.uk/2024/01/17/headlineDesktop.svg"
+								alt="Support fearless, independent journalism"
+							/>
+						</picture>
 					</h1>
```

Comment:
We have added an image tag inside an h1 tag here instead of simply adding an inline svg element to help with SEO. ... for reference adding an image with an alt text inside an h1 seems to bubble up the alt copy into the h1 copy with regards google referencing the page:

https://www.seroundtable.com/google-text-in-an-image-alt-attribute-h1-29007.html